### PR TITLE
fix: EthereumProvider: Appropriate try catch

### PR DESCRIPTION
**Pull request by Benjythebee**
When calling initialize of EthereumProvider, we can get the error "To use QR modal, please install @web3modal/standalone package"

The error thrown naively assumes the issue is with importing `@web3modal/standalone` when the issue could be from instancing the modal.
This PR adds a new try catch to make it easier to debug issues.

## Description

Please include a brief summary of the change.

## Type of change

- [ ] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Draft PR (breaking/non-breaking change which needs more work for having a proper functionality _[Mark this PR as ready to review only when completely ready]_)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How has this been tested?

This has not been tested however I could build the branch fine;

## Fixes/Resolves (Optional)

#2449

| Before       | After        |
| ------------ | ------------ |
| EthereumProvider.ts - Try catch| EthereumProvider.ts - one more Try catch |

## Checklist

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: providers/ethereum-provider/src/EthereumProvider.ts [hunk 1]
```diff
@@ -486,17 +486,25 @@ export class EthereumProvider implements IEthereumProvider {
...
       }
+      if (Web3modalClass) {
+        try {
+          this.modal = new Web3modalClass({
+            walletConnectVersion: 2,
+            projectId: this.rpc.projectId,
+            standaloneChains: this.rpc.chains,
+            ...this.rpc.qrModalOptions,
+          });
+        } catch {
```
Comment (ganchoradkov) [inline providers/ethereum-provider/src/EthereumProvider.ts]:
```suggestion
        } catch(e) {
      this.signer.logger.error(e);
```
Could we also log the error itself?

**Comment by Benjythebee** [comment]
Accepted suggestion, we should definitely log the error.

**Comment by ganchoradkov** [comment]
> Accepted suggestion, we should definitely log the error.

Awesome! One last thing, could you please run `prettier` as it seems to fail in CI and we'll be good to merge 💯

**Comment by Benjythebee** [comment]
Prettier ran!

**Review by ganchoradkov — APPROVED** [review]
Awesome! 💯

**Pull request merged**